doc/data/datasets.rst

```diff
@@ -0,0 +1,27 @@
+Datasets
+========
+
+
+deep_qa.data.dataset
```

Comment:
This title seems odd.  But we can worry about this later, when we're to the point of cleaning up docs.